iib/workers/tasks/build.py

```diff
@@ -535,9 +536,14 @@ def _prepare_request_for_build(
         with set_registry_token(overwrite_from_index_token, from_index):
             from_index_resolved = _get_resolved_image(from_index)
             from_index_arches = _get_image_arches(from_index_resolved)
+            ocp_version = get_image_label(from_index_resolved, "com.redhat.index.delivery.version")
         arches = arches | from_index_arches
     else:
         from_index_resolved = None
+        ocp_version = None
```

Comment:
Why not default `ocp_version` to 'v4.5' here rather than adding another check later?

Reply:
what happens if the index image does not have the label above? Still, need the check below this right?

Reply:
You're right. Do you think we can make that line 
`ocp_version = get_image_label(from_index_resolved, 'com.redhat.index.delivery.version') or 'v4.5'`? and default here to 'v4.5' as well?

Reply:
+1 that makes sense!